cdc/processor/pipeline/puller.go

```diff
@@ -54,23 +54,24 @@ func (n *pullerNode) tableSpan(ctx cdcContext.Context) []regionspan.Span {
 	// start table puller
 	config := ctx.ChangefeedVars().Info.Config
 	spans := make([]regionspan.Span, 0, 4)
-	spans = append(spans, regionspan.GetTableSpan(n.tableID, config.EnableOldValue))
+	spans = append(spans, regionspan.GetTableSpan(n.tableID, true))
 
 	if config.Cyclic.IsEnabled() && n.replicaInfo.MarkTableID != 0 {
-		spans = append(spans, regionspan.GetTableSpan(n.replicaInfo.MarkTableID, config.EnableOldValue))
+		spans = append(spans, regionspan.GetTableSpan(n.replicaInfo.MarkTableID, true))
```

Comment:
The second parameter we pass to `regionspan.GetTableSpan` is always `true` now, should we remove this parameter and simplify this function.

Reply:
It's false in some tests, I'll see if I can remove it.